### DISCUSSION TITLE
MyBMWAuthError: HTTPStatusError: invalid_client

### DISCUSSION THREAD

**Discussion by fregger2023** [bimmerconnected/bimmer_connected#670] [Q&A]
Hi all,
I've been using bimmer_connected cli for a while now, thanks for this great tool. Today however it stopped working, so I upgraded to bimmer-connected-0.16.4 (I was still using 0.13.8).
The first run -/path_to_/bimmerconnected status user@adress "password" rest_of_world- gave me following:

MyBMWAuthError due to HTTPStatusError: Client error '429 Too Many Requests' for url 'https://customer.bmwgroup.com/gcdm/oauth/authenticate'
For more information check: https://httpstatuses.com/429
MyBMWAuthError: HTTPStatusError: Client error '429 Too Many Requests' for url 'https://customer.bmwgroup.com/gcdm/oauth/authenticate'
For more information check: https://httpstatuses.com/429

So I waited a while before trying again, but now it gives me:

MyBMWAuthError due to HTTPStatusError: invalid_client - Client authentication failed (e.g., login failure, unknown client, no client authentication included or unsupported authentication method)
MyBMWAuthError: HTTPStatusError: invalid_client - Client authentication failed (e.g., login failure, unknown client, no client authentication included or unsupported authentication method)

Is there anything I can do to get this working again?

Best regards :-)

**Comment by rikroe** [comment]
It is blocked now, as BMW enforces a captcha for rest_of_world as well.
Please follow #671
